core/base-service/redirector.spec.js

```diff
@@ -24,6 +24,15 @@ describe('Redirector', function() {
     expect(redirector(attrs).name).to.equal('VeryOldServiceRedirect')
   })
 
+  it('overrides the name', function() {
```

Comment:
This was necessary to avoid a name conflict when adding two redirects with the same base URL (it had caused a validation test to fail).

Reply:
I was going to ask about the `name` addition above in redirector but I get it now 👍